components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -182,7 +182,8 @@ def deploy_model(action, model_name, default_model_uri, canary_model_uri, canary
         print('Sample test commands: ')
         print('# Note: If Istio Ingress gateway is not served with LoadBalancer, use $CLUSTER_NODE_IP:31380 as the ISTIO_INGRESS_ENDPOINT')
         print('ISTIO_INGRESS_ENDPOINT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath=\'{.status.loadBalancer.ingress[0].ip}\')')
-        print('curl -X GET -H "Host: ' + url.sub('', model_status['status']['url']) + '" $ISTIO_INGRESS_ENDPOINT')
+        host, path = url.sub('', model_status['status']['url']).split("/", 1)
```

Comment:
model_status['status']['url'] is like `http://flowers-sample.kubeflow.example.com/v1/models/flowers-sample` now

Reply:
Maybe add this as s comment.

Reply:
agree, done